spec/config_spec.rb

```diff
@@ -156,4 +156,41 @@
       end
     end
   end
+
+  describe "#ignored_environment?" do
+    describe "warnings" do
+      context "when 'ignore_environments' is set and 'environment' isn't" do
+        it "prints a warning" do
+          out = StringIO.new
+          config = described_class.new(logger: Logger.new(out))
```

Comment:
This two lines are the same for all tests here:
```ruby
out = StringIO.new
config = described_class.new(logger: Logger.new(out))
```
Should you move them to `let` block inside `describe "warning" do`?